server/webserver.go

```diff
@@ -912,6 +912,21 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				return
 			}
 
+			for _, o := range orchestrators {
```

Comment:
Can we add a test for this update (`cliserver_test.go` might be a good place for the test to go)?